src/FrontDoor/FrontDoor.Test/ScenarioTests/Common.ps1

```diff
@@ -60,4 +60,17 @@ function Assert-Tags($tags1, $tags2)
             throw "Tag content not equal. Key:$key Tags1:" +  $tags1[$key] + "Tags2:" + $tags2[$key]
         }
     }
+}
+
+<#
+.SYNOPSIS
+Sleep in record mode only
+#>
+function SleepInRecordMode ([int]$SleepIntervalInSec)
```

Comment:
@pichandwork this function shouldn't be needed -- you can use the existing `Wait-Seconds` function to do the same thing